src/OrchardCore.Modules/OrchardCore.Apis.GraphQL/AdminMenu.cs

```diff
@@ -22,9 +22,9 @@ public Task BuildNavigationAsync(string name, NavigationBuilder builder)
             }
 
             builder
-                .Add(S["Configuration"], NavigationConstants.AdminMenuConfigurationPosition, design => design
+                .Add(S["Configuration"], NavigationConstants.AdminMenuConfigurationPosition, configuration => configuration
                     .AddClass("menu-configuration").Id("configuration")
-                    .Add(S["GraphiQL"], S["GraphiQL"].PrefixPosition(), deployment => deployment
+                    .Add(S["GraphiQL"], S["GraphiQL"].PrefixPosition(), graphiQl => graphiQl
```

Comment:
graphQL

Reply:
graphiQl is fine here. Though, the changes you are requesting are not an issue.

 if it's just about renaming some variables we are getting again, out of scope. This can be done in a different PR.

Reply:
In fact I think those are all `menuItem`.

Reply:
If that makes any sense here some example that would still work : 

```C#
            builder
                .Add(S["Configuration"], NavigationConstants.AdminMenuConfigurationPosition, navigationItemBuilder => navigationItemBuilder
                    .AddClass("menu-configuration").Id("configuration")
                    .Add(S["GraphiQL"], S["GraphiQL"].PrefixPosition(), navigationItemBuilder => navigationItemBuilder
                        .Action("Index", "Admin", new { area = "OrchardCore.Apis.GraphQL" })
                        .Permission(Permissions.ExecuteGraphQL)
                        .LocalNav()
                    )
                );
```

So, not sure renaming these variables will have any consequence.

Reply:
I think we should have simply have used something like "nib" as a convention just like we often use "x => x.something"

Reply:
These are all builder. I think explicitly naming thing would help when nesting menu items. I tried to make the name relevant while being short. But I don’t know that full naming is really necessary.

Reply:
Only if you need to make mutations on the parent navigationItemBuilder from a child. Which probably never happens in OC.

Reply:
Yes, but these are builders so you will never need to use them other than for calling methods on them.

Reply:
Anyways, while it is an interesting discussion, we digress. 😄